conf/st2.conf.sample

```diff
@@ -13,7 +13,7 @@ logging = conf/logging.conf
 
 [api]
 # List of origins allowed
-allow_origin = http://localhost:3000  # comma separated list allowed here.
+allow_origin = http://127.0.0.1:3000  # comma separated list allowed here.
```

Comment:
Yes, it turned out to be an annoyance and took me about half an hour to figure out what happened.